recipes/calib/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export C_INCLUDE_PATH=${PREFIX}/include
+export LIBRARY_PATH=${PREFIX}/lib
+export CPATH=${PREFIX}/include
+
+make
```

Comment:
Is there a `make test` target that you can run?

Reply:
Looks like the answer is "no".